build/release/README.md

```diff
@@ -0,0 +1,63 @@
+# Releasing jQuery
+
+This document describes the process for releasing a new version of jQuery. It is intended for jQuery team members and collaborators who have been granted permission to release new versions.
+
+## Prerequisites
+
+Before you can release a new version of jQuery, you need to have the following tools installed:
+
+- [Node.js](https://nodejs.org/) (latest LTS version)
+- [npm](https://www.npmjs.com/) (comes with Node.js)
+- [git](https://git-scm.com/)
+
+## Release Process
+
+1. Ensure all milestoned issues/PRs are closed, or reassign to new milestone.
+2. Verify all tests are passing in [CI](https://github.com/jquery/jquery/actions).
+3. Run any release-only tests, such as those in the [`test/integration`](../../test/integration/) folder.
+4. Ensure AUTHORS.txt file is up to date (this will be verified by the release script).
+  - Use `npm run authors:update` to update.
+5. Create draft blog post on blog.jquery.com; save the link before publishing. The link is required to run the release.
+	- Highlight major changes and reason for release.
+	- Add contributor list generated in the below release script.
+	- Add HTML from the `changelog.html` generated in the below release script.
+6. Run a dry run of the release script:
+
+```bash
+BLOG_URL=https://blog.jquery.com/... npm run release -- -d
+```
+7. If the dry run is successful, run the release script:
+
+```bash
+BLOG_URL=https://blog.jquery.com/... npm run release
+```
+
+This will run the pre-release script, which includes checking authors, running tests, running the build, and cloning the CDN and jquery-dist repos in the `tmp/` folder.
+
+It will then walk you through the rest of the release process: creating the tag, publishing to npm, publishing release notes on GitHub, and pushing the updated branch and new tag to the jQuery repo.
+
+Finally, it will run the post-release script, which will ask you to confirm the files prepared in `tmp/release/cdn` and `tmp/release/dist` are correct before pushing to the respective repos. It will also prepare a commit for the jQuery repo to remove the release files and update the AUTHORS.txt URL in the package.json. It will ask for confirmation before pushing that commit as well.
+
+9. Once the release is complete, publish the blog post.
+
+## Stable releases
+
+Stable releases have a few more steps:
+
+1. Close the milestone matching the current release: https://github.com/jquery/jquery/milestones.
```

Comment:
...and create one for the next patch release if one already doesn't exist? I guess it will most of the time...

Reply:
I think that's covered in the first step above:

> Ensure all milestoned issues/PRs are closed, or reassign to a new milestone.

Reply:
There's an edge case possible where we just fix everything for a release and avoid creating a new one and then there's no milestone ready for new tickets. It's easy to create one when needed, though, so maybe it's good as-is.

Reply:
I see